installer/targets/robot/install.bash

```diff
@@ -1,2 +1,16 @@
+#Do not require sudo
+if [ ! -f /etc/sudoers.d/sudo-no-password ]
```

Comment:
I don't know if this is save

Reply:
Well, you are right that modifying the sudoers file is quit tricky. But I think it is less tricky once automated than people modifying these files themselves.

Hasn't gone wrong yet at least here ;-)

Feel free to try on your PC.